cerberus/kubernetes/client.py

```diff
@@ -316,10 +316,11 @@ def check_master_taint(master_nodes):
 
 def process_master_taint(master_nodes, iteration, iter_track_time):
     schedulable_masters = []
-    if iteration % 10 == 1:
-        check_taint_start_time = time.time()
-        schedulable_masters = check_master_taint(master_nodes)
-        iter_track_time['check_master_taint'] = time.time() - check_taint_start_time
+    if len(master_nodes) > 0:
+        if iteration % 10 == 1:
+            check_taint_start_time = time.time()
+            schedulable_masters = check_master_taint(master_nodes)
```

Comment:
Might need to pass the label here in case the user passes a different label than what we are expecting. 
Think this part will need to change: https://github.com/cloud-bulldozer/cerberus/blob/5a762ff331ac87fae4ef5ee1b77dc35438e9db5c/cerberus/kubernetes/client.py#L306

Reply:
fixed :+1: